api/main.go

```diff
@@ -115,21 +117,28 @@ func main() {
 		server := fireball.NewApp(routes)
 		server.ErrorHandler = controllers.ErrorHandler
 
+		lock := lock.NewDynamoLock(session, cfg.DynamoLockTable(), time.Minute*5)
```

Comment:
Slightly confused here --- we initialize `lock` here and pass it to each daemon service. Does this mean the same `lockID` is used for everything?

Reply:
no, that is a param is passed into `lock.Acquire` and `lock.Release`